usr/src/uts/common/fs/zfs/arc.c

```diff
@@ -6847,11 +6888,10 @@ l2arc_write_buffers(spa_t *spa, l2arc_dev_t *dev, uint64_t target_sz)
 			    zio_t *, wzio);
 
 			write_asize += size;
+
 			/*
 			 * Keep the clock hand suitably device-aligned.
```

Comment:
Thank you for spotting this.  I've removed this comment and adjusted the comment near the code where the definition of asize got moved.